src/group_theory/quotient_group.lean

```diff
@@ -266,6 +266,28 @@ def equiv_quotient_of_eq {M N : subgroup G} [M.normal] [N.normal] (h : M = N) :
   right_inv := λ x, x.induction_on' $ by { intro, refl },
   map_mul' := λ x y, by rw map_mul }
 
+/-- Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →* B / (B' ⊓ B)` induced by the inclusions. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →+ B / (B' ⊓ B)` induced by the inclusions."]
+def quotient_map_of_le {A' A B' B: subgroup G}
+  [hAN : (A'.subgroup_of A).normal] [hBN : (B'.subgroup_of B).normal]
+  (h' : A' ≤ B') (h : A ≤ B) :
+  quotient (A'.subgroup_of A) →* quotient (B'.subgroup_of B) :=
+map _ _ (subgroup.inclusion h) $
+  by simp [subgroup.subgroup_of, subgroup.comap_comap]; exact subgroup.comap_mono h'
+
+/-- Let `A', A, B', B` be subgroups of `G`.
+If `A' = B'` and `A = B`, then the quotients `A / (A' ⊓ A)` and `B / (B' ⊓ B)` are isomorphic. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`.
+If `A' = B'` and `A = B`, then the quotients `A / (A' ⊓ A)` and `B / (B' ⊓ B)` are isomorphic."]
+def quotient_equiv_of_eq {A' A B' B: subgroup G}
```

Comment:
This name is very confusing in the presence of `equiv_quotient_of_eq` above. Isn't `quotient_equiv_of_eq` trivial to implement in terms of `quotient_equiv_of_eq` by just substituting `h` and `h'`?

Reply:
Yes, I think something like `by unfreezingI { cases h'; cases h; refl }` proves it. I digress though because this is actually Mario's code: https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/Quotients.20of.20equivalent.20things.20are.20equivalent/near/234962412

He suggested that it would be better to construct the data normally.

Reply:
Maybe `equiv_quotient_of_eq_subgroups`? Not sure if that's better

Reply:
What happens if you try to define these as ```
quotient_equiv_of_eq $ begin
  rw h,
  rw h',
end
```

Reply:
Do you mean `equiv_quotient_of_eq`?

`A'.subgroup_of A` has type `subgroup A` while `B'.subgroup_of B` has type `subgroup B`. Of course `A = B`, but I don't know if it's possible (or if we even should) rewrite this when supplying the arguments to `equiv_quotient_of_eq`.

Reply:
Do you mean equiv_quotient_of_eq?

A'.subgroup_of A has type subgroup A while B'.subgroup_of B has type subgroup B. Of course A = B, but I don't know if it's possible (or if we even should) rewrite this when supplying the arguments to equiv_quotient_of_eq.

Reply:
Ah, thanks, I missed that the types didn't match.